### ISSUE TITLE
Please PR some examples 

### ISSUE THREAD

**Issue by jncd** [pivotal-cf/docs-ksm#93]
Hello @eduardoroliveira

My docs PM rejected the story for PR#78. She is asking for the following:

> Could KSM PR in a full example for steps 2 and 4 of [Replicate KSM Images](https://docs-pcf-staging.cfapps.io/ksm/0-n/installing.html#replicate-images)? 
There is only one command in the examples for these steps and this might imply that you only have to run one of the commands when I think you have to run all of them.

It's a sensible request and will save customer confusion. Are you able to do this PR? 

TIA, Jane

**Comment by joeeltgroth** [comment]
We think this has been addressed in the latest docs. Closing

**Comment by jncd** [comment]
Hi @joeeltgroth 
I know a lot got addressed recently, but sadly not this one. I've drafted the following — does that work?
![Screen Shot 2020-10-05 at 9 52 43 AM](https://user-images.githubusercontent.com/16601462/95109186-1b4f5100-06f1-11eb-80c2-a6c4653116ec.png)
 I'm supposed to have an example to match the command syntax  — if users have to run all three commands, then the example needs to show all three.
